spec/client_spec.rb

```diff
@@ -95,6 +108,13 @@
         hosts = [{host: '127.0.0.1', port: '7000'}]
         expect{ RedisCluster::Client.new(hosts) }.to raise_error Redis::CommandError
       end
+
+      it "initializes single node pool when force_cluster is false" do
```

Comment:
Note that the original behavior has been preserved and that I've added a separate test for `force_cluster` to check its functionality.